src/main/plugin/iso19139.ca.HNAP/schematron/schematron-rules-multilingual.sch

```diff
@@ -516,11 +516,11 @@
 
 
     <!-- Keywords -->
-    <sch:rule context="//gmd:identificationInfo/*/gmd:descriptiveKeywords/gmd:MD_Keywords/gmd:keyword">
-      <sch:let name="missing" value="not(string(gco:CharacterString))
+    <sch:rule context="//gmd:identificationInfo/*/gmd:descriptiveKeywords">
+      <sch:let name="missing" value="not(string(gmd:MD_Keywords/gmd:keyword[1]/gco:CharacterString))
```

Comment:
I don't believe this will work.

Previously the rule was for all //gmd:identificationInfo/*/gmd:descriptiveKeywords/gmd:MD_Keywords/gmd:keyword

Now the rule is only for the first gmd:keyword[1].

I'm guessing that it we were to add multiple keywords and the second+ keyword have issues. i.e. missing or does not contain other language, then the validation will still pass.  We will need to test this.

Reply:
Previously the rule was set for the first keyword. Now we are pushing it back to "//gmd:identificationInfo/*/gmd:descriptiveKeywords" level. So it will be a list. In this case, I dont think it's useful to create a util method to integrator thru all of them because if the first element is missing, this keyword list will be empty anyway. So it will only needed to check the first keyword